### PR TITLE
test: parametrized uploads

### PR DESCRIPTION
**Pull request by fstagni**

### CODE REVIEW

File: tests/.dirac-ci-config.yaml [hunk 1]
```diff
@@ -1,5 +1,8 @@
...
...vkt3xeg5HVQf2x/rNuhLTEhOMIFbbNOOqpO/WXYGyzFh/oVg7zb+DYKooYUCLV2TFdLaRCJQNpoZXlzq6O9ughPXZO4ruSEKrzv5MfoCU83OPtbmvvaBx6r2bNSxMo3FHH3yoftoJupCSV6sBZNZfWDNtz23xdcJ4vg4tNe2xv2pXjRsA1WqvVPciL3prDp46o4KPKF2UlSf8Q3J5oEmx+3/8HRV5zEhXYv4aX7ncVIPu0Y127HeAcDnmUMY7VLS7zEH35DC169mJC2pP+dGl6PY3aPF4yBXitHl+bsmoZo2Opatm6Z1fLyEW4d9oxYjCfD60TuhztSRK7XXz0SQv0csiiECR4z0xDorWeR5QOQIF69Gutyk+SK27KtHv7Y9zsN3S8KN/XPXZsgU6eJT+2cDsVMcgzCcNK219RIctxZw3i/rexz3XoiIC1PiVt3g2TzCAZz8KA3JnSlMp/+sJBhm+n0Y2sSalqUnBAVOhiFVDvBpGGCJdQwC3aQmSnwLhgrTfUNNGeGCuRvFBFPqvrTWCnGHKekWT179mNI6rmfsY/VP49ybqBe/8YbKqshZkzmTYR/SxBuOosHV3BQ314faayZBqgakEBDcQL4QZVixxJM1nVz4UBXLn1dBv3PHNHA1tdiWRoTZW8gbpVnhE3u2L3ET4bdf1MHZ+nF991laGmOua7ltoUdFi2KVq4Kf+8dWeBtsX4txnrUPL9K+g9ras/GAMX9OWB+5gkjY1zkX57JUGIVDToIjuN/9nveXkP0HClgb3KWNXY8QRtprECsuY3PlJ4FaxUIAAABqt0neAAHDJeiKAQBz3nq7PjANiwIAAAAAAVlaAAAAvgAEAAAAfXXY/nQAQQABAAAc/gAAAGQFVEZpbGUQcmVkdWNlZF9maWxlLmRzdA9Sb290IGV2ZW50IGRhdGEAAAACAAACSgAEAAAJIXXY/nQAOQABAAAE6gAAAGQFVFRyZWUEUmVmcxNSb290IHJlZmVyZW5jZSBkYXRhAAACmQAEAAAK6HXY/nQAQAABAAAHNAAAAGQFVFRyZWULRmlsZVJlY29yZHMTUm9vdCBkYXRhIGZvciBHYXVkaQAAAEsABAAAAAp12P50AEEAAQAAHbwAAABkBVRGaWxlEHJlZHVjZWRfZmlsZS5kc3QPUm9vdCBldmVudCBkYXRhAAEAAB4HdzWUAA==
```
Comment (chrisburr) [inline tests/.dirac-ci-config.yaml]: Why is this here? At a glance it just seems to be a file for testing, but if so why not generate random data on demand.

Committing random binary blobs makes me think of https://tukaani.org/xz-backdoor/ 😉

Comment (fstagni) [reply tests/.dirac-ci-config.yaml]: This is an LHCb DST file :-) with only 3 events in it (I need it for an LHCb test). I can move it to LHCb and here just put something simpler.
